webapi/src/main/scala/org/knora/webapi/messages/admin/responder/usersmessages/UsersMessagesADM.scala

```diff
@@ -618,8 +618,8 @@ case class UserADM(
    */
   def isSelf(identifier: UserIdentifierADM): Boolean = {
 
-    val iriEquals      = identifier.toIriOption.contains(id)
```

Comment:
@mpro7 Any clue why the equal signs after the variables are not indented anymore? In my opinion, this is one of the most important formatting settings.

Reply:
it needs to be at least more